File: Java Pattern Programs/Symbol Patterns/Pattern 12.java
class Pattern_75
{
    public static void main(String args[])
    {
        int size = 3;

        for (int i = size; i >= -size; i--)
        {
            for (int j = 1; j <= Math.abs(i); j++)
            {
                System.out.print(" ");
            }

            for (int k = size; k >= Math.abs(i); k--)
            {
                System.out.print("* ");
            }

            System.out.println();
        }
    }
}

File: CSharpPatternPrograms/SymbolPatterns/Pattern 12.cs
using System;

class Pattern_75
 {
  public static void Main()
   {
    int size = 4;
    for (int i = size; i >= -size; i--)
     {
      for (int j = 1; j <= Math.Abs(i); j++)
       {
        Console.Write(" ");
       }
       for (int k = size; k >= Math.Abs(i); k--)
        {
         Console.Write("* "); // space after '*'
        }
     Console.WriteLine();
     }
     Console.ReadKey(true);
   }
 }
